contracts/mixnet/src/contract.rs

```diff
@@ -435,7 +440,15 @@ pub fn query(deps: Deps<'_>, env: Env, msg: QueryMsg) -> Result<QueryResponse, C
     Ok(query_res?)
 }
 
-fn migrate_contract_state_params(deps: DepsMut<'_>) -> Result<(), ContractError> {
+fn update_epoch_duration(deps: DepsMut<'_>) -> Result<(), ContractError> {
+    let mut epoch = crate::interval::storage::current_epoch(deps.storage)?;
+    epoch.update_duration(600);
```

Comment:
Should we be changing  length of the current, existing, epoch to 10min for this test -> is it not going to cause some havoc for current epoch rewarding?
Why not instead force it to transition to a new epoch of length 10min?

Reply:
Its a good idea, but I don't see how we can do without overcomplicating this, since we'd need to update rewarded sets, distribute rewards and snapshot, not to mention mess with how epochs advance, as it can't advance while its in progress by design. With this change the validator-api will see that epoch needs advancing, and will just advance it, doing all the usual steps.

I'd suggest we test how this goes on QA, and if something horrible happens we'll think it through some more